src/main/java/com/google/sps/data/Event.java

```diff
@@ -15,31 +15,101 @@
 package com.google.sps.data;
 
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import com.google.gson.Gson;
 
 public class Event {
-  private final Long id;
+  private final String id;
   private final String title;
-  private final String date;
+  private final String description;
   private final List<String> tags = new ArrayList<String>();
+  private final String date_range;
+  private final String time_range;
```

Comment:
Now that I have seen how gcalender is being called I really think that storing numbers (Unix timestamp) is better than formatted dates. Using strings introduces problems like incorrectly formatted times, incompatible formats and unnecessary parsing, while storing numbers reduces the number of conversions (on user entry -> we can show an error, on display -> robust conversion).